package.json

```diff
@@ -61,11 +63,12 @@
     "flux": "2.1.1",
     "fuse.js": "2.6.1",
     "get-video-id": "2.1.4",
+    "globby": "3.0.1",
```

Comment:
What about using the [latest version](https://www.npmjs.com/package/globby), which is `6.1.0`?